packages/internal/test/test-utils.js

```diff
@@ -161,3 +162,99 @@ test('forever', async t => {
   }
   t.is(count, 4, 'count is expected');
 });
+
+/**
+ * @template T
+ * @param {AsyncIterable<T>} stream
+ * @param {Array<T>} output
+ * @param {number} [breakAfter]
```

Comment:
This is like `maxItems`, right? It's counting units of whatever the iterator is iterating over.